qutebrowser/browser/webkit/webkittab.py

```diff
@@ -781,6 +783,23 @@ def _on_frame_created(self, frame):
     def _on_contents_size_changed(self, size):
         self.contents_size_changed.emit(QSizeF(size))
 
+    @pyqtSlot()
+    def init_focustools(self):
+        """Initialize focustools.js in the main frame."""
```

Comment:
Since we started to have a `window._qutebrowser` on QtWebKit now, that should probably be a generic `_init_js` like with `WebEngineTab` here.